pkg/server/grpc.go

```diff
@@ -157,7 +163,9 @@ func (s *gRPCServer) FindMalwareInfo(c context.Context, r *pb.MalwareRequest) (*
 		}
 
 		for malware := range malwares {
-			if trim {
+			if isContainer {
+				malware.CompleteFilename = cntnrPathPrefixRegexObj.ReplaceAllString(malware.CompleteFilename, "")
```

Comment:
Since we want to remove a prefix, we might as well just find the end of the prefix and cut a slice out of `malware.CompleteFilename` (preventing extra string allocation)